stories/selectv2.stories.js

```diff
@@ -105,6 +107,8 @@ export const Rounded = () => (
       <Select variant="rounded" onChange={null} />
       <Title>With scrollbar (more than 4 items)</Title>
       <CustomSelect variant="rounded" opts={optionsWithoutSearchBar} />
+      <Title>1000 items</Title>
+      <CustomSelect variant="rounded" opts={generateOptions(1000)} />
```

Comment:
![image](https://user-images.githubusercontent.com/75977494/125790019-8d34575f-2a7a-4538-971e-bd3a4680591d.png)

It seems that some items are duplicated, seems to be a bug introduced when switching to virtualized list, could you please check ?

Reply:
Another bug is that we not scroll anymore to the selected entry when changing the focus from an item to another one

Reply:
It also break the scroll on keyboard navigation, icons are blinking on keyboard navigation.
I will check that.

Reply:
Additionally the scrollbar is invisible when there are 1000 items, I guess that's because the scroll handle is too small. To me we should add a min-height for the scroll-handle to provide context to the user about the scroll context in large documents.
For example here is the min height of default chrome scroll handle :
![image](https://user-images.githubusercontent.com/75977494/125792241-5569c53e-18e0-440d-99fa-f20db58ca3c3.png)